### DISCUSSION TITLE
recipe for target 'test' failed

### DISCUSSION THREAD

**Discussion by alexandrinapanfil** [vhive-serverless/vHive#245] [Q&A] [answered]
if I try to run 'make test' the following error appears: 
Makefile:46: recipe for target 'test' failed
make: *** [test] Error 1
because a lot of go dependencies cannot be found.  I am running this on a single node cluster in CloudLab with the vHive-cluster-quickstart profile. I don't have any experience with go so I do not understand how exactly to fix it.

**Comment by ustiugov** [comment, marked as answer]
hi @alexandrinapanfil , can you please describe all the steps that you followed and what you are trying to achieve?

**Reply by alexandrinapanfil** [reply]
I'm trying to do an evaluation of the artifact for a course at university and wanted to see how the performance metrics get collected (I saw that the metrics get dumped into bench.csv). I followed all of the steps in quickstart on setting up a single node cluster. Functions deploy and get invoked successfully. If I try to run tests with 'make test' the error appears. 
also the installed go version is 1.15 if it helps.

**Reply by ustiugov** [reply]
thank you. The testing environment is slightly different from the environment after following the quickstart guide. If you need to reproduce the results in the paper particularly, you should follow the instructions in the artifact that is the appendix in the PDF inside the artifact instead of the quickstart guide.

**Reply by alexandrinapanfil** [reply]
sorry, could you give me a link/name of this "PDF inside the artifact"?

**Reply by ustiugov** [reply]
ok, I think that there might be a confusion. What **exactly** are you trying to reproduce. Please provide more details.

**Reply by alexandrinapanfil** [reply]
I have to reproduce the same evaluation your team did, as described in paragraph 6 in the paper.

**Reply by ustiugov** [reply]
ok, here is what you need:
https://zenodo.org/record/4545584
there is a PDF file in the ZIP archive that has instructions at the back of it.

**Reply by alexandrinapanfil** [reply]
ok, I didn't know about this. thanks a lot!
